src/java/main/org/apache/zookeeper/server/quorum/Learner.java

```diff
@@ -193,21 +198,18 @@ void request(Request request) throws IOException {
     /**
      * Returns the address of the node we think is the leader.
      */
-    protected InetSocketAddress findLeader() {
+    protected QuorumServer findLeader() {
```

Comment:
Egah, this is a change from an earlier iteration that I forgot to remove. Fixed now.